packages/@okta/vuepress-theme-prose/assets/css/layouts/_authorizationLayout.scss

```diff
@@ -48,7 +47,7 @@
     }
   }
   input[type="submit"]:disabled  {
-    cursor: default;
+    cursor: not-allowed;
```

Comment:
Made a personal call here, that when the button is disabled, we should show the disabled cursor.